dataland-frontend/src/frameworks/BaseFrameworkRegistryImports.ts

```diff
@@ -0,0 +1,7 @@
+// <--WARNING--> THIS FILE IS AUTO-GENERATED BY THE FRAMEWORK-TOOLBOX AND WILL BE OVERWRITTEN
+import { type BaseFrameworkDefinition } from "@/frameworks/FrameworkDefinition";
+import gdv from "@/frameworks/gdv/BaseFrameworkDefinition";
+
+export const FrameworkDefinitions: Record<string, BaseFrameworkDefinition<object>> = {
+  gdv: gdv,
```

Comment:
Is this a list where the frameworks created using the toolbox would be added?

Reply:
Yes! (But this is also auto-generated)